django/db/models/fields/related.py

```diff
@@ -1314,7 +1314,7 @@ class ForeignObjectRel(object):
 
     # Reverse relations are always nullable (Django can't enforce that a
     # foreign key on the related model points to this model).
-    null = True
+    null = False
```

Comment:
Okay, I added a separate commit to keep `null=True` on `ForeignObjectRel` -- will squash before merging if this is fine.